sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -416,7 +416,7 @@ def read(self, verify_frame_type=0):
             offset = frame_header[4]
             frame_type = frame_header[5]
             if verify_frame_type is not None and frame_type != verify_frame_type:
-                raise ValueError(
+                _LOGGER.debug(
                     f"Received invalid frame type: {frame_type}, expected: {verify_frame_type}"
```

Comment:
We want to use printf style for logging for perf reasons :) Basically we want to hold back formatting this string till the very last minute (or when there is actually a problem). f-string or any other will evaluate it even if there is no exception